packages/runtime/src/templates/edge/function-runtime.ts

```diff
@@ -0,0 +1,23 @@
+import type { Context } from 'https://edge.netlify.com'
```

Comment:
I've split this into a separate entrypoint, because functions don't need custom matcher handling or support for advanced middleware - but _do_ want trailing slash handling unlike middleware